src/lib/translators/mongo.ts

```diff
@@ -231,16 +231,14 @@ function transformPivot(step: PivotStep): Array<MongoStep> {
 
 /** transform an 'replace' step into corresponding mongo steps */
 function transformReplace(step: ReplaceStep): MongoStep {
+  const branches: Array<MongoStep> = [];
```

Comment:
I suggest destructuring `rep` into `[oldval, newval]` and use `map`

```typescript
const branches = step.to_replace.map(([oldval, newval]) => ({
    case: { $eq: [$$(step.search_column), oldval] },
    then: newval,
}));
```

Reply:
Thank you for the suggestion !